diff -crB st2/config.def.h st/config.def.h
*** st2/config.def.h	2018-12-21 16:33:11.996609932 -0600
--- st/config.def.h	2018-12-21 16:29:50.613468514 -0600
***************
*** 5,11 ****
   *
   * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
   */
! static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
  static int borderpx = 2;
  
  /*
--- 5,11 ----
   *
   * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
   */
! static char *font = "Inconsolata:pixelsize=18:antialias=true:autohint=true";
  static int borderpx = 2;
  
  /*
***************
*** 83,89 ****
  unsigned int tabspaces = 8;
  
  /* bg opacity */
! unsigned int alpha = 0xcc;
  
  /* Terminal colors (16 first used in escape sequence) */
  static const char *colorname[] = {
--- 83,89 ----
  unsigned int tabspaces = 8;
  
  /* bg opacity */
! unsigned int alpha = 0xd9;
  
  /* Terminal colors (16 first used in escape sequence) */
  static const char *colorname[] = {
***************
*** 132,138 ****
   * 6: Bar ("|")
   * 7: Snowman ("☃")
   */
! static unsigned int cursorshape = 2;
  
  /*
   * Default columns and rows numbers
--- 132,138 ----
   * 6: Bar ("|")
   * 7: Snowman ("☃")
   */
! static unsigned int cursorshape = 4;
  
  /*
   * Default columns and rows numbers
